SETUP/INSTALL.md

```diff
@@ -28,6 +28,11 @@ Ubuntu system package names.
 * MySQL - php-mysql
 * zip - php-zip
 
+### Composer
+PHP package dependencies are managed using [Composer](https://getcomposer.org/).
+You need a recent version of Composer installed to manage the relevant PHP
```

Comment:
are there any known limits on which versions are allowed?

Reply:
Not that I know of. TEST has the version included with Ubuntu 20.04, but any 1.x and 2.x version should work. We aren't using any of the new 2.x features (and if we did, the JSON file will still work with 1.x).